pkgs/by-name/no/notion-app/package.nix

```diff
@@ -5,9 +5,7 @@
   unzip,
 }:
 let
-  info =
-    (builtins.fromJSON (builtins.readFile ./info.json))."${stdenvNoCC.targetPlatform.system}"
-      or (throw "notion-app: unsupported system ${stdenvNoCC.targetPlatform.system}");
```

Comment:
why this change?

Reply:
Because nixpkgs-update will fail if I use `throw`.
see `https://github.com/nix-community/nixpkgs-update/issues/468#issuecomment-2742539063`

Reply:
Fair enough, thanks.